_articles/fr/2023-11-02-atomic-design.md

```diff
@@ -0,0 +1,286 @@
+---
+contentType: article
+lang: fr
+date: 2023-11-02
+slug: atomic-design
+title: "Atomic Design : pour des interfaces modulaires et évolutives"
+excerpt: Explorez l'approche de l'Atomic Design et ses bénéfices pour vos développements d'interfaces. Les principes de modularité, flexibilité et maintenabilité sont des éléments-clés pour vous guider vers le succès.
+categories:
+    - javascript
+keywords:
+    - design system
+    - atomic design
+    - modularité
+    - react
+    - frontend
+authors:
+    - iregaibi
+---
+Dans cet article, je vais vous présenter le concept d'Atomic Design en détaillant ses avantages et en fournissant des exemples concrets de cette approche. Je vais également aborder l'application de cette méthodologie en proposant des exemples d'utilisation, tel que pratiqué dans nos projet au sein du [Studio Eleven Labs](https://eleven-labs.com/conception-d-application).
+
+## Qu'est-ce que l'Atomic Design
+Créé par Brad Frost en 2013, L'Atomic Design est tout d'abord une approche de conception de systèmes d'interface utilisateur et de design d'interaction, plus communément appellés [Design System](https://blog.eleven-labs.com/fr/pourquoi-creer-design-system/).
```

Comment:
also add a link from Design System article to this one as well: to do in a separated PR @fpasquet